lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -649,6 +661,82 @@ export abstract class ClientApplication {
         }
     }
 
+    /**
+     * Clears local cache for the current user then opens a popup window prompting the user to sign-out of the server
+     * @param logoutRequest 
+     */
+    async logoutPopup(logoutRequest: EndSessionRequest): Promise<void> {
```

Comment:
This needs to not be marked as `async` to properly mitigate the blocking performed by Safari.

Reply:
(it should/can still return a Promise)

Reply:
Done